### DISCUSSION TITLE
[Feature] Implement the manual trace API for Java

### DISCUSSION THREAD

**Discussion by CoderYellow** [apache/skywalking#10455] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

I have checked the code, the manual API described [here](https://skywalking.apache.org/docs/skywalking-java/v8.14.0/en/setup/service-agent/java-agent/application-toolkit-trace/) is not implemented. We need something like [brave](https://github.com/openzipkin/brave/tree/master/brave) . Because bytecode level instrumention is not suitable for framework like micronaut which aims to support graalvm.

### Use case

https://github.com/openzipkin/brave/tree/master/brave

### Related issues

https://github.com/micronaut-projects/micronaut-tracing/issues/260

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
If you love brave, please go ahead and use it. 

SkyWalking's manual APIs still require agent installed. It is not a SDK distribution.
